lib/stages/ReadGain.hpp

```diff
@@ -97,6 +97,8 @@ class ReadGain : public kotekan::Stage {
 
     /// Directory path where gain files are
     std::string _gain_dir_frb;
+    /// UT1-UTC value
+    std::float _UT1_UTC_val;
```

Comment:
Can you remove these extra lines that aren't used in ReadGain.hpp, and .cpp.

Reply:
Done!